src/components/organisms/rma-return-product-table/index.tsx

```diff
@@ -0,0 +1,109 @@
+import clsx from "clsx"
+import React from "react"
+import Button from "../../fundamentals/button"
+import MinusIcon from "../../fundamentals/icons/minus-icon"
+import PlusIcon from "../../fundamentals/icons/plus-icon"
+import TrashIcon from "../../fundamentals/icons/trash-icon"
+import Table from "../../molecules/table"
+
+type RMAReturnProductsTableProps = {
+  order: any
+  itemsToAdd: any[]
+  handleToAddQuantity: (value, index) => void
+  handleRemoveItem: (index) => void
+  quantities: any
+}
+
+const extractPrice = (prices, order) => {
```

Comment:
Small thing but you should properly use the displayAmount func from utils. Since not all currencies should be divided by 100 😄